sale_commission/models/account_move.py

```diff
@@ -36,6 +38,25 @@ def post(self):
 
     def recompute_lines_agents(self):
         self.mapped("invoice_line_ids").recompute_agents()
+        
+    @api.model
+    def fields_view_get(
+        self, view_id=None, view_type="form", toolbar=False, submenu=False
+    ):
+        res = super(AccountMove, self).fields_view_get(
```

Comment:
Add a docstring here commenting why doing this way:
```suggestion
        """Inject in this method the needed context for not removing other
        possible context values.
        """
        res = super(AccountMove, self).fields_view_get(
```